packages/bus/src/Brokers/RMQEventBus.spec.ts

```diff
@@ -0,0 +1,560 @@
+// eslint-disable-next-line max-classes-per-file
+import { RMQEventBus } from './RMQEventBus';
+import { RundomString } from '../utils';
+import { ConnectionFactory } from '../Factories/ConnectionFactory';
+import {
+  bind,
+  Command,
+  Configuration,
+  Event,
+  EventBus,
+  EventHandler
+} from '@secbox/core';
+import { injectable } from 'tsyringe';
+import { ConfirmChannel, Connection } from 'amqplib';
+import { anything, instance, mock, reset, verify, when } from 'ts-mockito';
+// Temporary fix for https://github.com/NagRock/ts-mockito/issues/191
```

Comment:
The blank line is mandatory
```suggestion
import { anything, instance, mock, reset, verify, when } from 'ts-mockito';

// Temporary fix for https://github.com/NagRock/ts-mockito/issues/191
```